text/0063-doc-comments.md

```diff
@@ -0,0 +1,226 @@
+# Policy and Schema Doc Comments
+
+## Related issues and PRs
+
+- Reference Issues:
+- Implementation PR(s):
+
+## Timeline
+
+- Started: 2024-04-12
+
+## Summary
+
+Cedar should support documentation comments (doc comments) on both policy and schema elements.
+
+Doc comments are a refinement to our current comments, distinguished by a third `/` and restricted in where they can appear.
+These comments are designed specifically to support rendering documentation as HTML pages (or similar), but this RFC does not propose what the final rendered form should be.
+
+## Basic example
+
+A policy can be documented by placing a doc comment, indicated by `///` following the style of Rust, immediately before the policy and on scope conditions.
+
+```cedar
+/// Allow users to create list and see the lists they own.
+@id("policy0")
+permit (
+    principal,
+    /// The `GetLists` action only returns the lists owned by that users, so this
+    /// policy does not allow users to see lists created by others.
+    action in [Action::"CreateList", Action::"GetLists"],
+    resource == Application::"TinyTodo"
+);
+```
+
+Doc comments may also be placed immediately prior schema elements to document the purpose of entity types and actions and before each attribute for an entity type or action context.
```

Comment:
what about before `principal` and `resource` in an `appliesTo` block?

(I do acknowledge the comment in Drawbacks that we need not block this RFC on discussing additional positions for doc comments, and additional ones can be added later as a non-breaking change.  We can punt on my comment here if we want.)

Reply:
These would also be good additions. I wasn't sure where to draw the line for what to include in this RFC. Hopefully extra comments positions can be added later without another RFC.